.gitignore

```diff
@@ -57,4 +57,5 @@ GTAGS
 .mypy_cache/
 /.pants
 /.venv
-.tool-versions
\ No newline at end of file
+.tool-versions
+pants.dev.toml
```

Comment:
Eh, we're poor at this, but you really mean `/pants.dev.toml` - just that one, not ones deeper in the tree - who knows what may happen.

Reply:
Yeah `gitignore` is great and all, but really confusing as hell IMO when it comes to leading and trailing slashes

Reply:
Yeah `gitignore` is great and all, but really confusing as hell IMO when it comes to leading and trailing slashes